anesthetic/plot.py

```diff
@@ -686,23 +665,26 @@ def fastkde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
                                            color=['c'],
                                            facecolor=['fc'],
                                            edgecolor=['ec']))
+
+    if len(data_x) == 0 or len(data_y) == 0:
+        return np.zeros(0), np.zeros(0), np.zeros((0, 0))
```

Comment:
This will need to be covered by the tests, or removed if you don't think its necessary. Possibly by `samples[:0].plot.plot_kde_1d()`?

Reply:
Sorry, this is not a new introduction from me, but a reordering to have more consistency between the plotting functions. 

Is this needed? I was tempted to drop it, but can't claim to know why it was introduced in the first place... In what scenario do you have an empty column? 

I think this should be and is all handled higher um in `Samples.plot_2d`, see e.g. this line:
```python
                    if x in self and y in self and lkwargs['kind'] is not None:
```

I'll drop it. The tests don't cover it after all...

Reply:
Removed in [6d0d8a3](https://github.com/williamjameshandley/anesthetic/pull/212/commits/6d0d8a3b823849e405569a0ea01bf17016f06064).